app/views/workbooks/_workbook.html.haml

```diff
@@ -0,0 +1,7 @@
+%li
+  %span
+    = workbook.name
+    |
+    = link_to "Instructions", workbook_path(workbook.id)
+    |
+    = link_to "Download", workbook.attachment.url
```

Comment:
this ghetto menu is the fastest I can make and quite useful while constructing basic functions.